index.html

```diff
@@ -1,29 +1,183 @@
+                <i class="fa fa-share"></i>
+                <span class="btn-value">SHARE</span>
+              </section>
+              <i class="fa fa-ellipsis-h"></i>
+            </section>
+          </section>
+          <section class="channel-details">
+            <div class="channel">
+              <img class="channel-icon"src="images/channel.JPG" alt="Avatar">
+              <div class="channel-name">
+                <span style="font-size: 1;"><b>Microverse Projects</b></span>
+                <span class="subscribers"> 1.1M subscribers </span>
+              </div>
+              <div class="subscribe"><button class="subscribe-button">subscribe</button></div>
+            </div>
+          </section>
+          <section class="more-details">
+            <div class="video-description">
+              <span>FOR HONOR Full Movie Cinematic 4K ULTRA HD Samurai Vs Viking Vs Knight All Cinematics Trailers</span>
+            </div>
+            <div class="showmore-span">
+              <span class="show-more">
+                show more
+              </span>
+            </div>
+          </section>
+      </section>
+      <div class="border"></div>
+      <section class="comment">
+        <h3>
+          583 comments
+        </h3>
+        <div class="sort">
+          <i class="material-icons">sort</i>
+          <span>SORT BY</span>
+        </div>
+      </section>
+      </section>
+      <section class="sidebar">
+        <aside>
+          <div class="next">
+            <div class="autoplay">
+              <h3>
+                Up next
+              </h3>
+              <span>
+                AUTOPLAY
+              <i class="fa fa-toggle-on" aria-hidden="true"></i>
+              </span>
+            </div>
+            <section class="sidebar-videos">
+              <img class="thumbnail" src="images/thumbnail.JPG">
+              <div class="item-details">
+                <p class="item-title">Hardest games Ever to be played</p>
+                <p class="item-channel">Gamers Repubulic</p>
+                <p class="item-count">8M views • 15 Aug 2020</p>
+              </div>
+            </section>
+          </div>
+          <section class="sidebar-videos">
+            <img class="thumbnail" src="images/thumbnail.JPG">
+            <div class="item-details">
+              <p class="item-title">Hardest games Ever to be played</p>
+              <p class="item-channel">Gamers Repubulic</p>
+              <p class="item-count">8M views • 15 Aug 2020</p>
+            </div>
+          </section>
+          <section class="sidebar-videos">
+            <img class="thumbnail" src="images/thumbnail.JPG">
+            <div class="item-details">
+              <p class="item-title">Hardest games Ever to be played</p>
+              <p class="item-channel">Gamers Repubulic</p>
+              <p class="item-count">8M views • 15 Aug 2020</p>
+            </div>
+          </section>
```

Comment:
- [ ] **OPTIONAL** I see you used sections to divide your content, but please keep in mind that for semantic HTML, the `<section>` tag is used to contain - according to W3C's HTML documentation - "a thematic grouping of content, typically with a heading". Here you are using it to position correctly your elements, you can use `<div>` tag for that purpose. As a tip, sections are used for bigger content.